src/widgetresize/resizerstate.js

```diff
@@ -146,6 +141,23 @@ export default class ResizeState {
 		this.proposedHandleHostWidth = newSize.handleHostWidth;
 		this.proposedHandleHostHeight = newSize.handleHostHeight;
 	}
+
+	/**
+	 * Calculates a relative width of a `domResizeHost` compared to it's parent in percents.
+	 *
+	 * @protected
```

Comment:
No one else uses this method so it should be private.

```suggestion
	 * @private
```